client/src/components/Logo.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import Logosvg from '../assets/logo.svg';
 import styled from '@emotion/styled';
+import { Link } from 'react-router-dom';
 
 const LogoDisplay = styled.img`
-  height: 200px;
-  width: 100%;
+  height: 125px;
+  margin-right: -40px;
```

Comment:
`margin-right` shouldn't be part of the logo. Instead, it's up to the parent to position the logo.

Reply:
Deleted Logo component since it is only used within the appbar. Positioning takes place in parent component now.